### DISCUSSION TITLE
Refacto de la section « Ressources utiles » en dessous des simulateurs

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2781] [Idées]
Les informations de cette section sont éclatées à plusieurs endroit de la base de code (`metada-scr`, `NextSteps.tsx`, ...). 

On pourrait considérer que l'affichage de ces liens pdf relève de l'éditorial au même titre des textes que l'on affiche sous les simulateurs, et faire un “opt-in” uniquement sur les pages sur lesquelles on veut afficher le lien `<NextStepPdfLink />`

On déplacerait donc cette logique sous forme de composant React, dans le même principe que les `explanations` ou les `seoExplanations`...

Cf https://github.com/betagouv/mon-entreprise/pull/2269#discussion_r962759996

**Comment by liliced** [comment]
Fait dans https://github.com/betagouv/mon-entreprise/commit/1b549adf8ac18a320d6faf88b0878b242d7230ba
